tests/core_tests/chaingen.h

```diff
@@ -752,7 +752,7 @@ struct push_core_event_visitor: public boost::static_visitor<bool>
     log_event("loki_blockchain_addable<cryptonote::checkpoint_t>");
     cryptonote::Blockchain &blockchain = m_c.get_blockchain_storage();
     bool added = blockchain.update_checkpoint(entry.data);
-    CHECK_AND_NO_ASSERT_MES(added == entry.can_be_added_to_blockchain, false, entry.fail_msg);
+    CHECK_AND_NO_ASSERT_MES(added == entry.can_be_added_to_blockchain, false, (entry.fail_msg.size() ? entry.fail_msg : "Failed to add checkpoint (no reason given)"));
```

Comment:
These seem unrelated to this PR (they seem fine, just checking that including them was intentional).

Reply:
Yea